app/models/ems_refresh.rb

```diff
@@ -51,6 +51,9 @@ def self.queue_refresh(target, id = nil, opts = {})
       h[e] << t unless e.nil?
     end
 
+    # Drop targets on EMSs which are using streaming refresh
+    targets_by_ems.reject! { |ems, _| ems.supports_streaming_refresh? }
```

Comment:
Some providers support it, but don't "currently" support it because of a setting flag...does this method handle that case?

Reply:
Yes we're checking the settings for if it is enabled not just if it is supported.